ceph.spec.in

```diff
@@ -159,7 +164,15 @@ BuildRequires:	gperftools-devel >= 2.6.1
 BuildRequires:	gperftools-devel >= 2.4
 %endif
 %endif
+%if 0%{with make_check}
 BuildRequires:  jq
+%if 0%{?fedora} || 0%{?rhel}
+BuildRequires:  pyOpenSSL%{_python_buildid}
+%endif
+%if 0%{?suse_version}
+BuildRequires:  python%{_python_buildid}-pyOpenSSL
+%endif
+%endif
```

Comment:
Please merge this `make_check` conditional block into the big `make_check` conditional block, below.

Reply:
i think they should belong to qa_run dependencies. so updated accordingly.

Reply:
It would be nice if this qa_run conditional block could be moved down so it's adjacent to the make_check conditional block.

Reply:
done.